src/recipe.rs

```diff
@@ -386,13 +389,16 @@ impl<'src, D> Recipe<'src, D> {
     })?;
     let mut path = tempdir.path().to_path_buf();
 
-    let extension = self.attributes.iter().find_map(|attribute| {
-      if let Attribute::Extension(extension) = attribute {
-        Some(extension.cooked.as_str())
-      } else {
-        None
-      }
-    });
+    let extension = self
```

Comment:
I think the original version of this is actually better, since we don't duplicate which attribute we're looking for.